src/renderer.js

```diff
@@ -73,11 +72,17 @@ function loadTasks() {
 
   // This will be done on loading
   existingTasks.forEach((task) => {
-    const taskDay = parseInt(task.whenTask.substring(8, 10), 10);
-    console.log("taskday: " + taskDay + " today + 1: " + (today + 1));
-    console.log("task.whenTask[6]: " + task.whenTask[6] + " month: " + month);
+    const taskDate = new Date(task.whenTask);
+    const taskDay = taskDate.getDate();
+    const taskMonth = taskDate.getMonth() + 1;
 
-    if (task.whenTask[6] == month && taskDay === today + 1) {
+    const isTomorrow =
```

Comment:
I realized (by natural circumstances because the end of January came) that my previous system for checking if its tomorrow didn't work so I had to figure out a different way.